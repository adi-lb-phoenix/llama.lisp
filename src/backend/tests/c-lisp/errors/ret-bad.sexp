(c-lisp
    (define ((main int))
        (ret 0 1)))
