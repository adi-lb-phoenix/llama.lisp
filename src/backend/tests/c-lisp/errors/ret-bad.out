Error in statement: ['ret', 0, 1]
CodegenError: Return statement can contain only 1 optional return value: ['ret', 0, 1]
